lib/mayaUsd/commands/PullPushCommands.cpp

```diff
@@ -416,14 +426,13 @@ MStatus DuplicateCommand::doIt(const MArgList& argList)
 
     MStatus    status = MS::kSuccess;
     MArgParser argParser(syntax(), argList, &status);
-    if (status != MS::kSuccess)
-        return status;
+    CHECK_MSTATUS_AND_RETURN_IT(status);
 
     status = parseUfePathArg(argParser, 0, _srcPath);
     if (status != MS::kSuccess)
         return reportError(status);
 
-    status = parseUfePathArg(argParser, 1, _dstPath);
+    status = parseUfePathArg(argParser, 1, _dstPath, true /*allowEmpty*/);
```

Comment:
I set new `allowEmpty` bool here to be able to accept as command input "" which means Maya "world" node.